ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-release-v10-presubmits.yaml

```diff
@@ -246,24 +246,6 @@ presubmits:
         resources: {}
       serviceAccountName: ci-operator
     trigger: (?m)^/test( | .* )unit,?($|\s.*)
-  - agent: kubernetes
```

Comment:
We should probably keep this job to ensure that upgrades for new patch releases on the 10.x branch still get tested.